dnsmasq-dhcpd-dynamic/smd.py

```diff
@@ -11,9 +11,13 @@ def getSMD(url):
     return data
 
 def getNID(c_data, xname):
+    if 'node_prefix' in os.environ:
```

Comment:
Is this generally how we want to set config options, or have we done it in different ways across different containers?